README.md

```diff
@@ -9,151 +9,151 @@ The Open-RMF platform for multi-fleet robot management.
+> Note: In order to install Open-RMF binaries, it is important to setup your computer to install ROS packages. This is a [prerequisite step](https://docs.ros.org/en/rolling/Installation/Ubuntu-Install-Debians.html#enable-required-repositories) when installing ROS 2 debian binaries.
 
 Install all non-ROS dependencies of Open-RMF packages,
 
 ```bash
 sudo apt update && sudo apt install \
-  git cmake python3-vcstool python3-pip curl \
+  python3-pip \
+  curl \
+  python3-colcon-mixin \
+  ros-dev-tools \
   -y
+
+sudo sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list'
+wget https://packages.osrfoundation.org/gazebo.key -O - | sudo apt-key add -
+
 python3 -m pip install flask-socketio fastapi uvicorn datamodel_code_generator
-sudo apt-get install python3-colcon*
 ```
 
-#### Install rosdep
-
-`rosdep` helps install dependencies for ROS packages across various distros. It can be installed with:
+`rosdep` helps install dependencies for ROS packages across various distros and will be installed along with `ros-dev-tools`.
+However, it is important to update it.
 
 ```bash
-sudo apt install python3-rosdep
-sudo rosdep init
+sudo rosdep init # run if first time using rosdep.
 rosdep update
 ```
 
-#### Download the source code
+Update `colcon mixin` if you have not done so previously.
 
-> Note all repositories now have a `galactic-devel` branch. New changes will be targeted for `ROS2 Humble`. If any change is compatible with `ROS2 galactic` they will be backported to the `galactic-devel` branch.
+```bash
+colcon mixin add default https://raw.githubusercontent.com/colcon/colcon-mixin-repository/master/index.yaml
+colcon mixin update default
+```
 
-Setup a new ROS 2 workspace and pull in the demo repositories using `vcs`,
+### Binary installation
+First follow the instruction in the [Setup section](#setup).
+Finally install Open-RMF debian packages for the `distro` of choice.
 
 ```bash
-mkdir -p ~/rmf_ws/src
-cd ~/rmf_ws
-wget https://raw.githubusercontent.com/open-rmf/rmf/main/rmf.repos
-vcs import src < rmf.repos
+sudo apt update && sudo apt install ros-<distro>-rmf-dev
 ```
 
-Ensure all ROS 2 prerequisites are fulfilled,
+**Note:** This will install all necessary debian packages to run Open-RMF except for those in [rmf_demos](https://github.com/open-rmf/rmf_demos) for reasons described in this [issue](https://github.com/open-rmf/rmf_demos/issues/166).
 
-you can subsutitute your distro name for `<your ros distro>`
+To build `rmf_demos` from source, first determine the version of `rmf_demos` that corresponds to the latest binary release for your distro.
+For example, if you installed the `humble` binaries of Open-RMF, the version of `rmf_demos` for this release can be found in the `rmf.repos` file in the `humble-release` branch of this repository, ie, [here](https://github.com/open-rmf/rmf/blob/591b0b9f34a021d81522a0af6f42ffcb5fc70293/rmf.repos#L5)
```

Comment:
```suggestion
For example, if you installed the `humble` binaries of Open-RMF, the version of `rmf_demos` for this release can be found in the `rmf.repos` file in the `humble-release` branch of this repository, ie, [here](https://github.com/open-rmf/rmf/blob/humble-release/rmf.repos#L5).
```

Reply:
Hmm I'd prefer the permalink as the command below references `2.0.3`. The version might be different if we always retrieve L5 from humle-release/rmf.repos.

Reply:
Gotcha, then perhaps we should rephrase it to

```suggestion
For example, if you installed the `humble` binaries of Open-RMF, the version of `rmf_demos` for this release can be found in the `rmf.repos` file in the `humble-release` branch of this repository, for example [here](https://github.com/open-rmf/rmf/blob/591b0b9f34a021d81522a0af6f42ffcb5fc70293/rmf.repos#L5).
```

since we are not providing the exact branch we are mentioning.

Reply:
Much better